### PR TITLE
feat(toolbar): Adds a banner and a badge for the PACER class action news

### PR DESCRIPTION
**Pull request by ERosendo**
This PR fixes https://github.com/freelawproject/recap/issues/367 by adding a badge to the extension icon in the toolbar and a new banner in the extension popup. 

Here's a screenshot of the new banner: 

image

Here's a GIF showing how the close icon works:

Screen Recording 2024-03-31 at 6 22 46 PM

I created this PR as a draft because we still need the URL for the 'Learn More' button :).

### CODE REVIEW

**Comment by ERosendo** [comment]
@mlissner I've applied the changes you requested. Here's a GIF showing how it works after the tweaks: 

Screen Recording 2024-04-02 at 1 10 49 PM

File: src/options.html
Comment (mlissner) [inline src/options.html:18]: Why put style code into the HTML instead of the CSS file?

**Review by mlissner — CHANGES_REQUESTED** [review]
Looks good. One question for you. 

We'll tweak the language once Jenifer and I know what it should be.

File: src/options.html
Comment (ERosendo) [reply src/options.html:18]: Sorry, I should've removed the "style" attribute while adding the new CSS classes to the banner. I often experiment with the style tag before moving things to classes. I'm cleaning those up now

**Comment by mlissner** [comment]
Great. Feel free to merge once complete, and then we'll do another PR once the language is ready for Firefox.

**Comment by ERosendo** [comment]
@mlissner the PR requires your approval before merging :).

**Review by mlissner — APPROVED** [review]

**Comment by ERosendo** [comment]
@mlissner Thanks!

**Pull request merged**
